src/app/courses/page.tsx

```diff
@@ -1,11 +1,101 @@
+'use client'
+
+import styled from '@emotion/styled'
 import { COURSES } from '@/data/courses'
+import { FontVariant, Color } from '@/app/theme'
 
 export default function Page() {
   return (
     <main>
-      {COURSES.map(({ title }, i) => (
-        <div key={i}>{title}</div>
-      ))}
+      <h1>Courses</h1>
+      <Sections>
+        {COURSES.map(({ code, title, description, editions }, i) => (
+          <>
```

Comment:
```suggestion
          <React.Fragment key={i}>
```
https://ko.legacy.reactjs.org/docs/fragments.html#keyed-fragments